chapter01/dxl/src/main/java/com/spring/services/HelloWorld.java

```diff
@@ -0,0 +1,29 @@
+package com.spring.services;
+
+
+public class HelloWorld implements EchoService {
+
+	private String message = "Hello, world!";
+
+	/*
+	 * I initially forgot this, but a bean needs a constructor with no arguments,
+	 * otherwise you get a runtime exception when initialising the object.
+	 */
+	public HelloWorld () {
+		super();
```

Comment:
I think `interface` doesn't allow constructors, so don't think we need `super` on the parent `interface`